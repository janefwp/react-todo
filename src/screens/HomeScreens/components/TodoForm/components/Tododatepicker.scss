div {
  margin-bottom: 10px;
  label {
    width: 30%;
  }
}
