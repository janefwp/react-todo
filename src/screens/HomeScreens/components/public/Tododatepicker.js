import React from 'react'
import DatePicker from "react-datepicker";
// import 'react-datepicker/dist/react-datepicker-cssmodules.css'
import './Tododatepicker.scss'
function Tododatepicker(props) {
    return (
        <div>
            
            <label className='datePickerLabel' htmlFor={props.name}>{props.label}</label>    
            <DatePicker selected={props.selected} onChange={props.onChange} className="datepicker"/> 
              
        </div>
    )
}

export default Tododatepicker
