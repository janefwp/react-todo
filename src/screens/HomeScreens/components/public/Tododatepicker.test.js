import React from 'react';
import renderer from 'react-test-renderer';
import Tododatepicker from './Tododatepicker'

it('renders correctly when there is no input', () => {
    const tree = renderer.create(<Tododatepicker />).toJSON();
    expect(tree).toMatchSnapshot();
  });