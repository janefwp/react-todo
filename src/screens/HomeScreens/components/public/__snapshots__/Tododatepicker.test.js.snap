// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there is no input 1`] = `
<div>
  <label
    className="datePickerLabel"
  />
  <div
    className="react-datepicker-wrapper"
  >
    <div
      className="react-datepicker__input-container"
    >
      <input
        className="datepicker"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        readOnly={false}
        type="text"
        value=""
      />
    </div>
  </div>
</div>
`;
